src/extensions/default/JavaScriptCodeHints/node/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "brackets-javascript-code-hints",
+  "dependencies": {
+    "acorn": "3.3.0",
```

Comment:
Acorn is used inside Tern by the Infer module. I agree with @ficristo, we will upgrade Acorn after a stable release.